addons/core/translations/resources/en-us.yaml

```diff
@@ -129,10 +129,15 @@ host-set:
         title: No Hosts Available
         description: No hosts available for selection.
   types:
-    static: Static
-    unknown: Unknown
-    aws: AWS
-    azure: Azure
+    static:
+      label: Static
+      help: Provide a list of hosts and host sets with static addresses
+    unknown:
+      label: Unknown
```

Comment:
Do we have not `help` for unknown, aws and azure?

Reply:
for host-sets, I don't think we do but I will check this further and address it if we need it!